dashboard/src/Piipan.Dashboard/Startup.cs

```
@@ -56,6 +59,22 @@ public void ConfigureServices(IServiceCollection services)
             services.AddDistributedMemoryCache();
             services.AddSession();
 
+            services.AddAuthorization(options => {
+                var authzPolicyOptions = Configuration
+                    .GetSection(AuthorizationPolicyOptions.SectionName)
+                    .Get<AuthorizationPolicyOptions>();
+
+                if (!(authzPolicyOptions is null))
```

I'm not familiar with this API yet, but it looks like it may fail open (i.e., grant access) if someone forgets to explicitly set the authorization policy in the app configuration. If so, would you consider re-writing so [it is more defensive and fails closed](https://blogs.keysight.com/blogs/tech/nwvs.entry.html/2020/05/20/fail_closed_failop-ZYAt.html)?

---

Great catch - reworked this a bit per your advice